UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.scss

```diff
@@ -1 +1,26 @@
 @use '../../../../manga-reader-common';
+
+
+.book-shadow.wide:before {
```

Comment:
We already have one in common, you should rework the selector there to accommodate for your single page addition.